### PR TITLE
Vet clinic database

### PR DESCRIPTION
**Pull request by RileyManda**
I  have completed the implementation of Vet clinic database schema according to the stipulated [requirements ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md): 🌱 

The following features have been implemented: 

- [x] Installed and setup postgresq in local environment
- [x] Generated the intial repo using [this template](https://github.com/microverseinc/curriculum-template-databases) 
- [x] Created a database named vet_clinic.
- [x] Created animals table using postgresql

- [x] **Executed the following queries on animals table**

- **Find all animals whose name ends in "mon".**

Screen Shot 2023-08-21 at 1 37 16 PM

- **Name of all animals born between 2016 and 2019.**

Screen Shot 2023-08-21 at 2 38 35 PM

- **List the name of all animals that are neutered and have less than 3 escape attempts.**

Screen Shot 2023-08-21 at 1 37 57 PM

- **List the date of birth of all animals named either "Agumon" or "Pikachu".**

Screen Shot 2023-08-21 at 1 38 04 PM

- **List name and escape attempts of animals that weigh more than 10.5kg**

Screen Shot 2023-08-21 at 1 38 10 PM

- **Find all animals that are neutered**

Screen Shot 2023-08-21 at 1 38 17 PM

- **Find all animals not named Gabumon.**

Screen Shot 2023-08-21 at 1 38 29 PM

- **Animals with a weight between 10.4kg and 17.3kg** 

Screen Shot 2023-08-21 at 1 38 37 PM

In addition, I have ensured that the project follows the following general requirements:

- Used the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- Used [this template](https://github.com/microverseinc/curriculum-template-databases) to generate the repo.

Please kindly review my implementation and suggest any changes.

Thank you 🙏

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -3,3 +3,12 @@
 INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Gabumon', '2018-12-15', 2, true, 8);
```
Comment (jerryowusu) [inline data.sql]: Kindly note that `Gabumon` was born on Nov 15th, 2018 as shown in the project requirements below. Please update accordingly. :+1: 

Screenshot from 2023-08-21 12-06-59

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @RileyManda 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

so-close


## Highlights
Used correct template :+1: 
Great effort so far :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RileyManda** [comment]
I have made the changes requested by the reviewer:Gabumon was born on Nov 15th, 2018

Screen Shot 2023-08-21 at 2 33 07 PM

Please kindly review my implementation and suggest any changes.

Thank you 🙏

**Review by jerryowusu — APPROVED** [review]
Hi @RileyManda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

weldone


## Highlights
Used correct template 👍
Great effort so far 👍


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**